packages/vsce/src/commands/inspectResourceCommandUtils.ts

```diff
@@ -85,10 +117,19 @@ async function loadResources(
   profileName: string
 ): Promise<IResourcesHandler> {
   const resourceContainer = new ResourceContainer<IResource>(resourceType);
-  resourceContainer.setCriteria([resourceName]);
+  if (resourceName) {
+    resourceContainer.setCriteria([resourceName]);
+  }
   resourceContainer.setProfileName(profileName);
   const resources: [Resource<IResource>[], boolean] = await resourceContainer.loadResources(session, regionName, cicsplex);
 
+
+  if (resources[0].length === 0 ) {
+    CICSLogger.error(CICSMessages.CICSResourceNotFound.message);
+    window.showErrorMessage(CICSMessages.CICSResourceNotFound.message);
```

Comment:
How do we feel about some context of what we're trying to do here? 
`CICSProgram MYPROG1 not found in region MYREG10`.

Might help users diagnose why their request isn't working?